### ISSUE TITLE
"Upgrade Required" when I connect to my server

### ISSUE THREAD

**Issue by quantumproducer** [okaybenji/100ng#1]
I cloned the code, installed packages, connected to :8080 and see a page with "Upgrade Required" text.

**Comment by okaybenji** [comment]
Hey, sorry, just now seeing this issue. I've not seen the "Upgrade Required" text. Are you still having this problem with the current version?

**Comment by quantumproducer** [comment]
Yep - just made a fresh checkout. Could this be an npm / node issue, perhaps?

```
~/100ng:.npm start -v
3.10.3
~/100ng:.npm start -verbose
npm info it worked if it ends with ok
npm verb cli [ '/usr/local/bin/node',
npm verb cli   '/usr/local/bin/npm',
npm verb cli   'start',
npm verb cli   '-verbose' ]
npm info using npm@3.10.3
npm info using node@v4.0.0
npm verb run-script [ 'prestart', 'start', 'poststart' ]
npm info lifecycle 100ng@0.0.7~prestart: 100ng@0.0.7
npm info lifecycle 100ng@0.0.7~start: 100ng@0.0.7

> 100ng@0.0.7 start /Users/me/100ng
> node scripts/server/server.js
```

**Comment by okaybenji** [comment]
Thanks, this is helpful! It looks like you're using an older version of Node (4.0.0) which does not support ES6. This code is written in ES6, so you'll need to upgrade Node to use it. I'm currently on 6.3.0. Would you be able to upgrade Node and try again?

**Comment by quantumproducer** [comment]
Weird, I'm running 

```
~/code/100ng:.node -v
v6.4.0
~/code/100ng:.npm -v
3.10.3
```

and I still see the message

**Comment by okaybenji** [comment]
Okay, I see the problem now. The Node server is not an HTTP server. It just keeps track of the multiplayer game. In order to load and play the client locally, you'll need to start up a web server and serve the files from the main project directory. Personally, I am using Brackets to host my local development HTTP server.

**Comment by okaybenji** [comment]
Oh, alternatively you can just open index.html in Chrome! It should still work.

(Note that the connection in scripts/socket.js is currently pointing to the live/production Node server, but you can comment out line 21 and uncomment line 20 to point to your local Node server.)

**Comment by okaybenji** [comment]
Hey @quantumproducer, have you had a chance to try this and see if it works for you?

**Comment by quantumproducer** [comment]
Hey @okaybenji , haven't had a chance yet. Thank you for the tip though! I will try asap.

**Comment by quantumproducer** [comment]
Hi @okaybenji - there is no scripts/socket.js . I found these (you can see where I added a line) in index.js:

```
// TODO: move these inside startGame (here for now for console debugging)
let players = {};
let player; // ourself/client avatar

(function startGame() {
  // const ws = new WebSocket('wss://138.68.12.151:8080'); // Digital Ocean server -- does NOT work over WSS
  // const ws = new WebSocket('wss://l00ng-okaybenji.c9users.io:8080'); // Cloud 9 server -- works over WSS
  const ws = new WebSocket('wss://localhost:8080');
```

**Comment by quantumproducer** [comment]
I did a pull - looks like I was a few commits behind, same thing though!
